spring-data-jdbc-plus-support/src/main/java/com/navercorp/spring/data/jdbc/plus/support/convert/SqlContext.java

```diff
@@ -53,20 +54,6 @@ private static Table getEntityTable(RelationalPersistentEntity<?> entity) {
 		return table;
 	}
 
-	@Override
-	public Column getIdColumn() {
-		return table.column(entity.getIdColumn());
-	}
```

Comment:
I removed some public methods immediately because they were also deleted [original source](https://github.com/spring-projects/spring-data-relational/commit/2a24738ad15c6f5fa8ad561f3be2ce7038e572e3#diff-9b258fdc9ffbb16dcc623ac4928ae2ac2163455cdd9833595e0ee3bcb3d94f50).

If you mind this changes, I will recover them and mark as deprecated.